pkgs/development/libraries/libvmaf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, autoconf, automake, intltool, libtool, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "libvmaf";
+  version = "1.3.15";
+
+  src = fetchurl {
+    url="https://github.com/Netflix/vmaf/archive/v1.3.15.tar.gz";
+    sha256="13ajbcidngjvgl0rr7l0mb43h651p5pqj3d1linrfk9c222b9fs3";
+  };
+
+  buildInputs = [ autoconf automake intltool libtool pkgconfig ];
+  doCheck = true;
+
+  installPhase = ''
+    make INSTALL_PREFIX=$out install
+  '';
```

Comment:
this is the default install phase, you can remove

Reply:
@jonringer Are you sure? Without passing that INSTALL_PREFIX as a parameter to make like that it installs in /usr/local

Reply:
I apologize, `make install` is the default installPhase. what I meant was
```
makeFlags = [ "INSTALL_PREFIX=${placeholder "out"}" ];
```